### PR TITLE
Update player.yml

### PR DESCRIPTION
**Pull request by FicelleYTB**

### CODE REVIEW

File: lua-docs/content/reference/player.yml [hunk 1]
```diff
@@ -22,13 +22,13 @@ properties:
                 value:Replace(newBlock)
             end
     - name: "ID"
-      type: "number"
```
Comment (XLegland) [inline lua-docs/content/reference/player.yml:25]: Standard Lua components do not start with a capital letter

File: lua-docs/content/reference/player.yml [hunk 2]
```diff
@@ -39,6 +39,9 @@ properties:
     - name: "Velocity"
       type: "Number3"
       description: "Player's velocity (speed + direction)."
+     - name: "Wave"
+      type: "Boolean"
+      description: "Set the player animation wave to true or false."
```
Comment (XLegland) [inline lua-docs/content/reference/player.yml:44]: Be more precise about what the property does, like "when set to true, the Player starts waving, it stop when it is set to false". Also do not forget to put brackets around `Player` so it links to the correct page, you could also add ` characters around code words like true.

File: lua-docs/content/reference/player.yml [hunk 3]
```diff
@@ -39,6 +39,9 @@ properties:
     - name: "Velocity"
       type: "Number3"
       description: "Player's velocity (speed + direction)."
+     - name: "Wave"
```
Comment (XLegland) [inline lua-docs/content/reference/player.yml:42]: Be careful with the indentation, the hyphen should be at the same column as the one on line 39.

**Review by XLegland — CHANGES_REQUESTED** [review]
Thanks for updating the documentation, there some changes to be done, but it is still great work 👍

**Review by XLegland — COMMENTED** [review]
Wave is a function, so it should be placed in the `functions` part

**Comment by FicelleYTB** [comment]
Ok

**Comment by aduermael** [comment]
This PR is outdated, closing.

**Pull request closed without merging**